migrate_pim/upgrade_major_version.rst

```diff
@@ -0,0 +1,36 @@
+How to upgrade to a major version?
+==================================
+
+We release a major version each year, these new major versions bring new features and larger changes to answer clients growing needs.
```

Comment:
Should we add "We release a major version each year, in February" ?

Reply:
> Should we add "We release a major version each year, in February" ?

Maybe it is better to avoid dates, in case the release date changes.

Reply:
I've add a note : To check the PIM version release date, check our article with the link to https://www.akeneo.com/knowledge-base/how-many-pim-versions-are-maintained-by-akeneo/